runtime/src/test/java/io/quarkiverse/loggingjson/providers/JsonProviderBaseTest.java

```diff
@@ -16,12 +28,52 @@
 
 abstract class JsonProviderBaseTest {
 
-    private static final JsonFactory jsonb = new JsonbJsonFactory();
-    private static final JsonFactory jackson = new JacksonJsonFactory();
-    private static final ObjectMapper mapper = new ObjectMapper();
+    private final JsonFactory jsonb = new JsonbJsonFactory(prettyPrint());
```

Comment:
I don't like the way the test is created. And the test should not format the output, because what are you then running your test against, not the output of the actual JsonFormatter.

Reply:
OK, fair enough.

I had tried to keep duplication to a minimum.

The `JsonGenerator` in `quarkus-logging-json` delegates "pretty printing" to their underlying JSON library and as such the _format_ for the pretty printed JSON differs between the two (line breaks, indenting and spacing). I added the `format(..)` method around the _expected_ value and related code so that the test could format the _exected_ value with the underlying JSON library. I've added a new commit removing that and duplicating the literal _expected_ values for each `JsonGenerator`.

This is _your_ project and I respect that. I missed being able to "pretty print" when using it so thought I'd try adding the missing implementation. I'm happy to adhere to your requirements, guidelines etc. I think having the missing implementation remains a useful addition.

Reply:
@SlyngDK Would you like me to make any further changes?